hadoop-ozone/interface-client/src/main/proto/OmClientProtocol.proto

```diff
@@ -145,6 +146,7 @@ message OMRequest {
   optional DBUpdatesRequest                  dbUpdatesRequest              = 53;
   optional FinalizeUpgradeRequest           finalizeUpgradeRequest         = 54;
   optional FinalizeUpgradeProgressRequest   finalizeUpgradeProgressRequest = 55;
+  optional PrepareRequest                   prepareRequest                 = 56;
```

Comment:
Can we rename to PrepareUpgradeRequest/PrepareUpgradeResponse?

Reply:
The design doc and flow diagram are a bit inconsistent with usage of PrepareRequest or PrepareUpgradeRequest. We were initially planning on calling it PrepareUpgradeRequest during the design phase, but later decided that the command is not actually specific to upgrades. It will also be used for downgrades, and it is really a more general operation to clear out the OM logs, without any upgrade/downgrade specific actions in it. We opted to keep the name generic to reflect this.

@avijayanhwx let me know if I missed anything here, and your current thoughts on the prepare vs. prepareUpgrade naming.

Reply:
+1 to @errose28's comment. We can keep it "prepare" as a generic operation that can be used during upgrade, downgrade as well as creating checkpoint for supporting "rollbacks" in the future.

Reply:
Okay, get it.